### PR TITLE
second times a charm

### PR DESCRIPTION
**Pull request by justinengland**
changes to allow proper secrets access for upload stages to run during build time. Second attempt, to get around phantom sub module changes in previous git branch.

### CODE REVIEW

File: .github/workflows/build-linux-arm64-installer.yml [hunk 1]
```diff
@@ -44,6 +44,16 @@ jobs:
...
 
+    - name: Test for secrets access
+      id: check_secrets
+      shell: bash
+      run: |
+        unset HAS_SECRET
+        if [ -n "$SECRET" ]; then HAS_SECRET='true' ; fi
+        echo ::set-output name=HAS_SECRET::${HAS_SECRET}
+      env:
+        SECRET: "${{ secrets.test_pypi_password }
```
Comment (cmmarslender) [inline .github/workflows/build-linux-arm64-installer.yml:55]:
```suggestion
        SECRET: "${{ secrets.TEST_PYPI_PASSWORD }
```


Pretty sure theyse are all caps

File: .github/workflows/build-linux-installer.yml [hunk 2]
```diff
@@ -78,6 +78,16 @@ jobs:
...
 
+    - name: Test for secrets access
+        id: check_secrets
+        shell: bash
+        run: |
+          unset HAS_SECRET
+          if [ -n "$SECRET" ]; then HAS_SECRET='true' ; fi
+          echo ::set-output name=HAS_SECRET::${HAS_SECRET}
+        env:
+          SECRET: "${{ secrets.test_pypi_password }
```
Comment (cmmarslender) [inline .github/workflows/build-linux-installer.yml:89]:
```suggestion
          SECRET: "${{ secrets.TEST_PYPI_PASSWORD }
```

**Review by cmmarslender — CHANGES_REQUESTED** [review]

**Review by cmmarslender — DISMISSED** [review]

File: .github/workflows/build-linux-arm64-installer.yml [hunk 3]
```diff
@@ -44,6 +44,16 @@ jobs:
...
 
+    - name: Test for secrets access
+      id: check_secrets
+      shell: bash
+      run: |
+        unset HAS_SECRET
+        if [ -n "$SECRET" ]; then HAS_SECRET='true' ; fi
+        echo ::set-output name=HAS_SECRET::${HAS_SECRET}
+      env:
+        SECRET: "${{ secrets.TEST_PYPI_PASSWORD }
```
Comment (cmmarslender) [inline .github/workflows/build-linux-arm64-installer.yml:55]:
```suggestion
        SECRET: "${{ secrets.TEST_PYPI_PASSWORD }"
```

**Review by cmmarslender — DISMISSED** [review]

**Review by cmmarslender — APPROVED** [review]

**Comment by arvidn** [comment]
This PR landed despite breaking the `build-linux-installer.yml` CI workflow.
I've corrected it here, https://github.com/Chia-Network/chia-blockchain/pull/6862
please review!

**Pull request merged**